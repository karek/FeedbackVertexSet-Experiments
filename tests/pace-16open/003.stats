53,89,11
