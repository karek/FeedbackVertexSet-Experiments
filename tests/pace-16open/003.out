10
36 49 27 17 47 23 4 41 14 6 
